src/particle.c

```diff
@@ -160,3 +160,63 @@ int reb_remove_by_id(struct reb_simulation* const r, int id, int keepSorted){
 	}
 	return success;
 }
+
+struct reb_particle reb_subtract_particles(struct reb_particle p1, struct reb_particle p2){
+    struct reb_particle p = {0};
+    p.x = p1.x - p2.x;
+    p.y = p1.y - p2.y;
+    p.z = p1.z - p2.z;
+    p.vx = p1.vx - p2.vx;
+    p.vy = p1.vy - p2.vy;
+    p.vz = p1.vz - p2.vz;
+    p.ax = p1.ax - p2.ax;
+    p.ay = p1.ay - p2.ay;
+    p.az = p1.az - p2.az;
+    p.m = p1.m - p2.m;
+    return p;
+}
+
+struct reb_particle reb_add_particles(struct reb_particle p1, struct reb_particle p2){
```

Comment:
I've been very inconsistent myself, but I try to give functions a name such as `reb_FILENAME_...`. Do you think `reb_particle_add_particles` or `reb_particle_operator_add` or `reb_particle_add` would make sense?